### DISCUSSION TITLE
Fixing visionos build errors

### DISCUSSION THREAD

**Discussion by RobPruzan** [callstack/react-native-visionos#86] [Q&A] [answered]
Kept getting 
```
xcodebuild: error: Unable to find a destination matching the provided destination specifier:
                { id:CEC64381-5C04-4B6F-995D-E435F09DB942 }

        Available destinations for the "<project-name>" scheme:
                { platform:macOS, arch:arm64, id:00006020-001C11340287401E }
                { platform:macOS, arch:x86_64, id:00006020-001C11340287401E }
                { platform:macOS, name:Any Mac }
```
when attempting to run ```pnpm visionos``` after completing readme instructions

And I fixed that by going to 
```Xcode > settings > Locations > Command Line Tools ```
And switching from Xcode 15.1 to Xcode-beta 15.2

**Comment by Saadnajmi** [comment, marked as answer]
Xcode 15.2 is the first official (not beta) version to have visionOS Support. You probably want to upgrade to that.

**Reply by RobPruzan** [reply]
Good idea 
EDIT: To note, I had 2 xcode installs, and it seems like 15.2 beta and non beta are the same builds (afaik). So the above fix/tip should still be relevant

**Reply by Saadnajmi** [reply]
Yep! Same build numbers for most of the included SDKs, so it seems that 15.2 was just a release for visionOS really
